lighthouse-plugin-ad-speed-insights/audits/ad-top-of-viewport.js

```diff
@@ -53,10 +54,13 @@ class AdTopOfViewport extends Audit {
    */
   static audit(artifacts) {
     const viewport = artifacts.ViewportDimensions;
-    const slots = artifacts.RenderedAdSlots
-        .filter((slot) => slot && slot.width > 1 && slot.height > 1)
-        .map((slot) =>
-          ({midpoint: slot.content[1] + slot.height / 2, id: slot.id}));
+    const slots = artifacts.IframeElements
+        .filter((slot) => slot.boxModel.width > 1 && slot.boxModel.height > 1
```

Comment:
FYI, for readability this could be

```
   artifacts.IframeElements.filter({boxModel) => boxModel.width > 1 && boxModel.height > 1)
       .filter(isGoogleAdsIframe)
```

Reply:
Moved the >1 check to the gatherer. Unable to use your suggested filter syntax due to index being interpreted as the second arg to isGPTIFrame().